src/python/grpcio/grpc/_compression.py

```diff
@@ -39,7 +39,7 @@ def create_channel_option(compression):
              int(compression)),) if compression else ()
 
 
-def augment_metadata(metadata, compression):
+def augment_metadata(metadata, compression) -> tuple:
```

Comment:
Do we need this type annotation?

Reply:
This annotation is breaking the CI https://source.cloud.google.com/results/invocations/d752270a-8bf6-43cc-9e2c-191f63da5672/targets/github%2Fgrpc%2Frun_tests%2Fpython_linux_opt_gevent%2Fpy27_gevent.test.unit._signal_handling_test.SignalHandlingTest/tests